README.md

```diff
@@ -80,12 +80,17 @@ To modify download information or change packaging information send a pull reque
 
 ## Forking/Redistributing/Custom builds policy
 
-Do whatever you want, we don't care. Just follow the license. If you have any questions about this feel free to ask in an issue.
+We don't care what you do with your fork as long as you do the following as a basic courtesy:
+- Follow the terms of the [license](LICENSE) (not just a courtesy, but also a legal responsibility)
+- Make it clear that your fork is not PolyMC and is not endorsed by or affiliated with the PolyMC project (https://polymc.org).
+- If you are distributing this fork, go through [CMakeLists.txt](CMakeLists.txt) and change PolyMC's API keys to your own or set them to empty strings (`""`) to disable them (this way the program will still compile but the functionality requiring to those keys will be disabled).
+
+If you have any questions or want any clarification on the above conditions please make an issue and ask us.
 
 Be aware that if you build this software without removing the provided API keys in [CMakeLists.txt](CMakeLists.txt) you are accepting the following terms and conditions:
  - [Microsoft Identity Platform Terms of Use](https://docs.microsoft.com/en-us/legal/microsoft-identity-platform/terms-of-use)
  - [CurseForge 3rd Party API Terms and Conditions](https://support.curseforge.com/en/support/solutions/articles/9000207405-curse-forge-3rd-party-api-terms-and-conditions)
-If you do not agree with these terms and conditions, then remove the associated API keys from the [CMakeLists.txt](CMakeLists.txt) file.
+If you do not agree with these terms and conditions, then remove the associated API keys from the [CMakeLists.txt](CMakeLists.txt) file by setting them to an empty string (`""`).
```

Comment:
Not about this PR, but since you're at it, can you add a new line here? Markdown is weird and isn't breaking line where it should, so it's glued to the CF ToS item :/

Reply:
Do you mean "it should" as in "it is intended to" because by markdown syntax being glued to the line above is correct.

Reply:
fixed it anyway

Reply:
> Do you mean "it should" as in "it is intended to" because by markdown syntax being glued to the line above is correct.

Yeah it's intended to, markdown is just weird with it's syntax sometimes